js/packages/candy-machine-ui/src/App.tsx

```diff
@@ -1,54 +1,63 @@
 import './App.css';
 import { useMemo } from 'react';
-
+import * as anchor from '@project-serum/anchor';
 import Home from './Home';
 
-import * as anchor from '@project-serum/anchor';
 import { clusterApiUrl } from '@solana/web3.js';
 import { WalletAdapterNetwork } from '@solana/wallet-adapter-base';
 import {
   getPhantomWallet,
+  getSlopeWallet,
   getSolflareWallet,
   getSolletWallet,
+  getSolletExtensionWallet,
 } from '@solana/wallet-adapter-wallets';
 
 import {
   ConnectionProvider,
   WalletProvider,
 } from '@solana/wallet-adapter-react';
-
 import { WalletDialogProvider } from '@solana/wallet-adapter-material-ui';
+
 import { ThemeProvider, createTheme } from '@material-ui/core';
-import { ConfettiProvider } from './confetti';
 
 const theme = createTheme({
   palette: {
     type: 'dark',
   },
 });
 
-const candyMachineId = process.env.REACT_APP_CANDY_MACHINE_ID
-  ? new anchor.web3.PublicKey(process.env.REACT_APP_CANDY_MACHINE_ID)
-  : undefined;
+const getCandyMachineId = (): anchor.web3.PublicKey | undefined => {
+  try {
+    const candyMachineId = new anchor.web3.PublicKey(
+      process.env.REACT_APP_CANDY_MACHINE_ID!,
+    );
 
-const fairLaunchId = process.env.REACT_APP_FAIR_LAUNCH_ID
-  ? new anchor.web3.PublicKey(process.env.REACT_APP_FAIR_LAUNCH_ID)
-  : undefined;
+    return candyMachineId;
+  } catch {
```

Comment:
please log that constructing the public key failed - it'll be helpful during debugging